src/app/components/LanguageSources.svelte

```diff
@@ -15,7 +17,8 @@
    * @return {void}
    */
   const handleAddSource = (): void => {
-    console.log("Add source button clicked");
+    addSource = true;
+    console.log("Add button clicked");
```

Comment:
Is this `console` needed or just for debugging?

Reply:
console is just for debugging

Reply:
Okay, I saw langaugeSource page is also written like this, I will remove the console in other places.

Reply:
See #137 for a follow-up on this disucssion.